docs/how-to/craft/manage-data-compatibility.rst

```diff
@@ -3,10 +3,59 @@
 Manage data compatibility
 =========================
 
+By defining an epoch in a snap's project file, snap authors can control how users
+receive a snap release when its data format is incompatible with older versions.
+
 
 Define an epoch for a snap
 --------------------------
 
+When a new snap release breaks data compatibility with older versions, snap authors can
+increment the value of the ``epoch`` key in the snap's project file. This will stop
+users of older versions from automatically refreshing to the newer release.
+
+By default, snaps have an epoch of ``0``. If a new version were to break data
+compatibility with snaps of this epoch, the new version's ``epoch`` key should be
+incremented as follows:
+
+.. code-block:: yaml
+    :caption: snapcraft.yaml
+
+    epoch: 1
+
+This will preserve the functionality of older versions until they can be safely
+transitioned to the newer epoch.
+
+
+Update a specific epoch
+-----------------------
+
+To push updates to users of a specific epoch, edit the value of the project's ``epoch``
+key to reflect the target epoch. For example, if you want to push an update to users
+of epoch 0 after releasing epoch 1, add the following to the project file in the
+revision:
```

Comment:
I'm not sure what "revision" means in this context.
```suggestion
of epoch 0 after releasing epoch 1, update your project file to include:
```

Reply:
"to include the newer epoch"?